server/lib/pbench/test/unit/test_process_tb.py

```diff
@@ -0,0 +1,283 @@
+import os
+import logging
+import pytest
+
+from pathlib import Path
+from typing import List
+
+from pbench.process_tb import ProcessTb, Results
+from pbench.test import tarball, bad_tarball, fixture_path
+
+
+class MockConfig:
+    TS = "run-1970-01-02T00:00:00.000000"
+
+    def get(self, section: str, option: str) -> str:
+        return ""
+
+
+class TestProcessTb:
+    def test_token(self, monkeypatch, test_logger):
+        def mock_get(self, section: str, option: str) -> str:
+            assert option in [
+                "pbench-receive-dir-prefix",
+                "put-token",
+            ], f"Unexpected configuration option, {section!r}/{option!r}, in mocked config.get()"
+            return fixture_path if option == "pbench-receive-dir-prefix" else ""
+
+        expected_error_msg = (
+            "No value for config option put-token in section pbench-server"
+        )
+        monkeypatch.setattr(MockConfig, "get", mock_get)
+        with pytest.raises(ValueError) as e:
+            ProcessTb(MockConfig, test_logger)
```

Comment:
This should be invoking the constructor, `MockConfig()`, not passing the class object.

Ditto lines 47 and 65.

Reply:
I don't know how I missed these

Reply:
It took me several passes before I found them.  😸

Reply:
Thanks a lot :)